tests/plugins/image2/manual/upcast.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 4.11.2, bug, 2506
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, image2, sourcearea
+
+1. Open developer console.
```

Comment:
Probably testing procedure could be simplified, e.g. by adding two buttons, which will be inserting empty `figure` and `figure > a` elements.